src/Object/Api/Mastodon/Account.php

```diff
@@ -92,37 +92,39 @@ class Account extends BaseEntity
 	 */
 	public function __construct(BaseURL $baseUrl, array $publicContact, Fields $fields, array $apcontact = [], array $userContact = [])
 	{
-		$this->id              = $publicContact['id'];
+		$this->id              = (string)$publicContact['id'];
 		$this->username        = $publicContact['nick'];
 		$this->acct            =
 			strpos($publicContact['url'], $baseUrl->get() . '/') === 0 ?
 				$publicContact['nick'] :
 				$publicContact['addr'];
 		$this->display_name    = $publicContact['name'];
 		$this->locked          = !empty($apcontact['manually-approve']);
+		$this->bot             = ($publicContact['contact-type'] == Contact::TYPE_NEWS);
+		$this->discoverable    = !$publicContact['unsearchable'];
+		$this->group           = ($publicContact['contact-type'] == Contact::TYPE_COMMUNITY);
 		$this->created_at      = DateTimeFormat::utc($publicContact['created'], DateTimeFormat::ATOM);
-		$this->followers_count = $apcontact['followers_count'] ?? 0;
-		$this->following_count = $apcontact['following_count'] ?? 0;
-		$this->statuses_count  = $apcontact['statuses_count'] ?? 0;
 		$this->note            = BBCode::convert($publicContact['about'], false);
 		$this->url             = $publicContact['url'];
 		$this->avatar          = $userContact['avatar'] ?? $publicContact['avatar'];
 		$this->avatar_static   = $userContact['avatar'] ?? $publicContact['avatar'];
 		// No header picture in Friendica
 		$this->header          = '';
 		$this->header_static   = '';
-		// No custom emojis per account in Friendica
-		$this->emojis          = [];
-		// No metadata fields in Friendica
-		$this->fields          = $fields->getArrayCopy();
-		$this->bot             = ($publicContact['contact-type'] == Contact::TYPE_NEWS);
-		$this->group           = ($publicContact['contact-type'] == Contact::TYPE_COMMUNITY);
-		$this->discoverable    = !$publicContact['unsearchable'];
+		$this->followers_count = $apcontact['followers_count'] ?? 0;
+		$this->following_count = $apcontact['following_count'] ?? 0;
+		$this->statuses_count  = $apcontact['statuses_count'] ?? 0;
 
 		$publicContactLastItem = $publicContact['last-item'] ?: DBA::NULL_DATETIME;
 		$userContactLastItem = $userContact['last-item'] ?? DBA::NULL_DATETIME;
 
 		$lastItem = $userContactLastItem > $publicContactLastItem ? $userContactLastItem : $publicContactLastItem;
-		$this->last_status_at  = $lastItem != DBA::NULL_DATETIME ? DateTimeFormat::utc($lastItem, DateTimeFormat::ATOM) : null;
+		$this->last_status_at  = $lastItem != DBA::NULL_DATETIME ? DateTimeFormat::utc($lastItem, 'Y-m-d') : null;
+
+		// No custom emojis per account in Friendica
+		$this->emojis          = [];
+		// No metadata fields in Friendica
```

Comment:
Please remove this since we do have metadata fields.

Reply:
Done